applications/NurbsBrepApplication/custom_python/add_custom_utilities_to_python.cpp

```diff
@@ -20,17 +20,192 @@
 #include "processes/process.h"
 #include "custom_python/add_custom_utilities_to_python.h"
 
-#include "spaces/ublas_space.h"
-#include "linear_solvers/linear_solver.h"
+#include "nurbs_brep_application_variables.h"
+#include "custom_utilities/NodeCurveGeometry3D.h"
+#include "custom_utilities/NodeSurfaceGeometry3D.h"
 
 namespace py = pybind11;
 
 namespace Kratos {
 namespace Python {
 
-void  AddCustomUtilitiesToPython(py::module& m)
+void AddCustomUtilitiesToPython(py::module& m)
 {
+    using namespace pybind11::literals;
+
+    using Vector3D = Kratos::array_1d<double, 3>;
```

Comment:
If you typedef this then I recommend to name it `Array3` or `Array3D`, this is usually used

Otherwise it can be confused with the ublas/amatrix Vector